platform/ethereum/domain.go

```diff
@@ -1,14 +1,33 @@
 package ethereum
 
 import (
-	CoinType "github.com/trustwallet/blockatlas/coin"
+	"strings"
+
+	"github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/pkg/address"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
 	"github.com/trustwallet/blockatlas/pkg/errors"
 	"github.com/trustwallet/blockatlas/pkg/logger"
 	"github.com/trustwallet/blockatlas/platform/ethereum/ens"
-	AddressEncoder "github.com/trustwallet/ens-coincodec"
+	enscoincodec "github.com/trustwallet/ens-coincodec"
```

Comment:
remove alias

just use 
coincodec.ToString() ..

Reply:
It's ens-coincodec, but in ens-coincodec.ToString() auto formatter introduces spaces... :(
<img width="492" alt="image" src="https://user-images.githubusercontent.com/13562139/83768803-8293c080-a67f-11ea-9bae-5b182b420ec6.png">

Reply:
coincodec.ToString()

Reply:
changed